minutus/build.rs

```diff
@@ -74,16 +73,28 @@ fn main() -> Result<()> {
 
     let out_dir = env::var("OUT_DIR")?;
     let build_config_copy = Path::new(&out_dir).join("build_config.rb");
-    std::fs::copy(
-        &env::current_dir()?.join("build_config.rb"),
+    fs::copy(
+        env::current_dir()?.join("build_config.rb"),
         &build_config_copy,
     )?;
 
     let do_link = env::var("CARGO_FEATURE_LINK_MRUBY").is_ok();
+    let do_download = match env::var("CARGO_FEATURE_MRUBY_DIR") {
+        Err(_) => true,
+        _ => match env::var("MINUTUS_MRUBY_DIR") {
+            Ok(dir) if dir.trim().is_empty() => true,
```

Comment:
I cannot imagine `mruby_dir` usecases where `MINUTUS_MRUBY_DIR` is not set (or empty string is set).
Why not `bail!` if `mruby_dir` is enabled but `MINUTUS_MRUBY_DIR` is not set?